src/store/transactions/fungible-token-transfer.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { useAtomValue } from 'jotai/utils';
+import { ProviderWithWalletAndRequestToken } from '../../../tests/state-utils';
+
+import { makeFungibleTokenTransferState } from '@store/transactions/fungible-token-transfer';
+import { setupHeystackEnv } from '../../../tests/mocks/heystack';
+
+describe('makeFungibleTokenTransferState', () => {
```

Comment:
```suggestion
describe(makeFungibleTokenTransferState.name, () => {
```
editor rename tools can then update it if it changes name

Reply:
atoms don't have names like functions do, but they have an optional `debugLabel` which i've updated.

my IDE also searches for string references when doing a rename function